tgui/packages/tgui/interfaces/PreferencesMenu/antagonists/antagonists/stowawaychangeling.ts

```diff
@@ -0,0 +1,18 @@
+import { Antagonist, Category } from '../base';
+import { multiline } from 'common/string';
+import { CHANGELING_MECHANICAL_DESCRIPTION } from './changeling';
+
+const Stowaway_Changeling: Antagonist = {
+  key: 'stowawaychangeling',
+  name: 'Stowaway Changeling',
+  description: [
+    multiline`
+      A Changeling that found its way onto the shuttle
+      unbeknownst to the crewmembers on board.
```

Comment:
```suggestion
      A Changeling surprise replacement one of the
      crewmembers during their transit to the station.
```
my wording is poor but i think implying they killed the person instead would be more flavorful, up to you though

Reply:
Methinks the original wording can still be interpreted as such. That was my original intention anyhow, to allow it to either be interpreted as a head ling or a full fledged ling